test/utils.test.js

```diff
@@ -12,7 +12,7 @@ var mochaAsync = fn => {
 describe("Utils", function() {
   describe("greetings", function() {
     it("Says Hello World", function() {
-      assert.equal(utils.greetings("World"), "Hello World 👋👋");
+      assert.equal(utils.greetings("World"), "Hello everyone 👋👋");
```

Comment:
```suggestion
      assert.equal(utils.greetings("World"), "Hello World 👋👋");
```
Please accept my suggestion